index.bs

```diff
@@ -536,6 +536,19 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 
 # Triggering Algorithms # {#trigger-algorithms}
 
+<h3 algorithm id="noising-data-fields">Noising trigger data</h3>
+
+This section defines how to noise [=attribution trigger/trigger data=] and
+[=attribution trigger/event source trigger data=].
+
+To <dfn>noise trigger data</dfn> given an integer |input|, integer |maxData|, and double |noiseRate|,
+where |input| must be between 0 (inclusive) and |maxData| (exclusive) and |noiseRate| must be
+between 0 and 1 (both inclusive), run the following steps:
```

Comment:
wdyt about moving input arg requirements into asserts, similar to: https://fetch.spec.whatwg.org/#concept-request-add-range-header